scripts/symbolize.py

```diff
@@ -94,6 +95,8 @@ def get_elf(self, elf_or_uuid):
         if not elf_or_uuid.endswith('.elf'):
             elf_or_uuid += '.elf'
         for d in self._dirs:
+            if d.endswith(elf_or_uuid) and glob.glob(d):
```

Comment:
Maybe I misunderstood what you was trying to do with `glob.glob(d)`, but in case if it is a verification of file existence, maybe it's better to use `d.is_file()` (more obvious)?

Reply:
Correct. That would be `os.path.isfile(d)`, but then we would need `import os`, I'm not sure it is worth it. Would you prefer that? I'm fine with both ways.

Reply:
@jforissier sorry for suggesting `d.is_file()`, found it's possible to use in python 3 only (where OOP-friendly `pathlib` exists). I'm fine with both ways also, but IMHO usage of `os.path.isfile(d)` will help easily understand what this code supposed to do (at least for me :) )

Anyway:
`Reviewed-by: Igor Opaniuk <igor.opaniuk@linaro.org>`